text/1132-default-strict-resolver.md

```diff
@@ -0,0 +1,145 @@
+---
+stage: accepted
+start-date: 2025-08-13T00:00:00.000Z
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams:
+  - framework
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/1132
+project-link:
+suite: 
+---
+
+# Default Strict Resolver
+
+## Summary
+
+This RFC proposes shipping a built-in strict resolver as the default in Ember applications, replacing the current `ember-resolver` package. The strict resolver uses explicit module registration through `import.meta.glob('...', { eager: true })` (or manually) instead of dynamic string-based lookups, providing better tree-shaking, build-time optimization, and improved developer experience with static analysis tools.
```

Comment:
The point of this is that you can pass in a source of modules -- this isn't limited to `import.meta.glob`